alma/views/templates/hook/payment_button_deferred.tpl

```diff
@@ -26,3 +26,6 @@
     </p>
     {include file="modules/alma/views/templates/hook/_partials/deferred.tpl" plans=$plans}
 </div>
+{if $first}
```

Comment:
first ?

Reply:
@Francois-Gomis
It's the first gateway of the list. I don't know why it made like this but yes, we need to improve this.
It seams to display only one block for inpage.

Reply:
we can together investigate this "$first", it's probably to have only 1 Block but I think it's important to understand why and comment/rename this variable